packages/select/src/components/ui/GroupedMultiSelect.tsx

```diff
@@ -114,7 +121,13 @@ export const GroupedMultiSelect = <TData extends {}>({
             flexDirection={"row"}
             flexGrow={1}
           >
-            <Text size={"small"} tabIndex={-1}>
+            <Text
+              size={"small"}
+              tabIndex={-1}
+              style={{
```

Comment:
Let `resolveSelectedTextColor` return the whole style object instead, so if there is no color, it can return undefined.

No need to create the object if no styles are set.

Reply:
Or use Text color prop?

Reply:
Ah, right, I mistook it for the Box color prop, which we removed :)

Reply:
Fixed!